### PR TITLE
Bump version to 0.4.0 for release

### PR DESCRIPTION
**Pull request by donv**
Remove duplicate LICENSE paragraph in README
Removed obsolete TODO and reset_gem files

@Serabe Could you have a quick look?  If it looks OK, I would like to go for release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-use with JRuby. 
+[RMagick](https://github.com/rmagick/rmagick) is a Ruby binding to ImageMagick
+and GraphicsMagick.  **RMagick4J** implements ImageMagick functionality and the
+C portions of RMagick for use with JRuby.
 
 ## Authors
 
-This project was written by Thomas Palmer, Sergio Rodríguez Arbeo, and Thomas
-Enebo with lots of help from the JRuby community.
+This project was written by Thomas Palmer, Sergio Rodríguez Arbeo, Thomas
```
Comment (Serabe) [inline README.md]: Can you please remove the `Rodríguez` here? It is no longer there in my name. Thank you so much.

**Comment by Serabe** [comment]
👍

**Comment by donv** [comment]
OK, going for release.

**Comment by Serabe** [comment]
Thank you, @donv!

**Pull request merged**